api/Navigator.json

```diff
@@ -2365,6 +2365,112 @@
           }
         }
       },
+      "unregisterProtocolHandler": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "38"
```

Comment:
The data for `registerProtocolHandler` says "13" here. Weren't they shipped together?

Reply:
I've confirmed they weren't implemented together, https://trac.webkit.org/changeset/119482/webkit added `unregisterProtocolHandler` long after `registerProtocolHandler` shipped.

It was then completed and shipped in https://bugs.chromium.org/p/chromium/issues/detail?id=392016 and that's consistent with shipping in Chrome 38.

Also, https://mdn-bcd-collector.appspot.com/tests/api/Navigator/unregisterProtocolHandler is very simple, not much that can go wrong with that test.